packages/adapter-cloudflare/index.js

```diff
@@ -71,15 +71,15 @@ export default function (options = {}) {
  */
 function get_routes_json(builder, assets, { include = ['/*'], exclude = ['<all>'] }) {
 	if (!Array.isArray(include) || !Array.isArray(exclude)) {
-		throw new Error(`routes.include and routes.exclude must be arrays`);
+		throw new Error('routes.include and routes.exclude must be arrays');
 	}
 
 	if (include.length === 0) {
-		throw new Error(`routes.include must contain at least one route`);
+		throw new Error('routes.include must contain at least one route');
 	}
 
 	if (include.length > 100) {
-		throw new Error(`routes.include must contain 100 or fewer routes`);
+		throw new Error('routes.include must contain 100 or fewer routes');
```

Comment:
I think it's easier to always write a backtick, but harder to read. When I see a backtick I look to see what variable is used inside and that ends up being a waste of brain cycles when it turns out there is none. I personally like this rule as it makes it faster to mentally parse the code. We can have `pnpm format` do it automatically so that it's not hard to remember. But yeah, if you want to modify a string to insert a variable you will need to convert it to backticks. I think that happens less than reading the code, which is why I tend to optimize for reading rather than modifying